src/main/java/com/ericsson/ei/jsonmerge/MergeHandler.java

```diff
@@ -100,10 +100,12 @@ public String mergeObject(String id, String mergeId, RulesObject rules, String e
         try {
             // lock and get the AggregatedObject
             String aggregatedObject = getAggregatedObject(id, true);
+            /* For test prints
             System.out.println("\n\n\n");
             System.out.println("should merge at: " + mergePath);
             System.out.println("BEFORE:");
             System.out.println(aggregatedObject);
+            */
```

Comment:
Same comment here. Remove it if its not going to be used.

Reply:
Already done.